webapp/packages/core-notifications/src/NotificationsItem/NotificationsItem.tsx

```diff
@@ -24,7 +26,7 @@ export const NotificationsItem = observer<Props>(function Notification({ notific
     return <Custom notification={notification} {...notification.extraProps} />;
   }
 
-  let closeDelay = 0;
+  let closeDelay = MIN_CLOSE_DELAY;
 
   if (notification.type !== ENotificationType.Error && !notification.persistent && notification.autoClose !== false) {
     closeDelay = FADE_TIMEOUT;
```

Comment:
it's possible that we now broke this behavior (that some notifications must be closed instantly and some not)
and we have two different delays 

now, all notifications will be closed after `MIN_CLOSE_DELAY` even when we want them to be open

check `useActivationDelay` in `Snackbar`